src/kagglehub/config.py

```diff
@@ -84,17 +91,11 @@ def get_log_verbosity():
     return DEFAULT_LOG_LEVEL
 
 
-def _get_kaggle_credentials_file():
+def _get_kaggle_credentials_file() -> str:
     return os.path.join(_get_kaggle_credentials_folder(), CREDENTIALS_FILENAME)
 
 
-def _get_kaggle_credentials_folder():
+def _get_kaggle_credentials_folder() -> str:
     if CREDENTIALS_FOLDER_ENV_VAR_NAME in os.environ:
         return os.environ[CREDENTIALS_FOLDER_ENV_VAR_NAME]
     return DEFAULT_KAGGLE_CREDENTIALS_FOLDER
-
-
-@dataclass
```

Comment:
Had to move this class to the top of the file to allow using it in type hints above.